src/app/views/query-response/adaptive-cards/AdaptiveCard.tsx

```diff
@@ -154,6 +154,30 @@ class AdaptiveCard extends Component<IAdaptiveCardProps> {
   }
 }
 
+function onPivotItemClick(item?: PivotItem) {
+  if (!item) { return; }
+  const key = item.props.itemKey;
+  if (key) {
+    trackTabClickEvent(key);
+  }
+}
+
+function trackTabClickEvent(tabKey: string) {
+  switch (tabKey) {
+    case 'templateJSON': {
+      telemetry.trackEvent(
+        eventTypes.TAB_CLICK_EVENT,
+        {
+          ComponentName: componentNames.JSON_SCHEMA_TAB
+        });
+      break;
+    }
+    default: {
+      break;
+    }
+  }
+}
+
```

Comment:
There may be no need for the switch case at this point. The telemetry tab click  event can be called immediately